libexec/python/utils.py

```diff
@@ -57,13 +56,13 @@ def add_http(url,use_https=True):
     prefix = "https://"
     if use_https == False:
         prefix="http://"
-    
+
     # Does the url have http?
-    if re.search('^http*',url) == None:
+    if not url.startswith('http'):
```

Comment:
(and note in python 3 it is not under [urllib.parse](https://docs.python.org/3.0/library/urllib.parse.html))